executor/point_get.go

```diff
@@ -424,8 +424,13 @@ func EncodeUniqueIndexValuesForKey(ctx sessionctx.Context, tblInfo *model.TableI
 			str, err = idxVals[i].ToString()
 			idxVals[i].SetString(str, colInfo.FieldType.Collate)
 		} else {
+			// Since this func is only used batch point get, the idxVals should be exactly equals to value in
```

Comment:
If a truncated error or an overflow error is thrown when converting the type of \`idxVal[i]\` to the type of \`colInfo\`, the \`idxVal\` does not exist in the \`idxInfo\` for sure.